### PR TITLE
layer.conf: update layer to scarthgap series

### PR DESCRIPTION
**Pull request by adhudase**
Change-Id: I2e0beb508efb016f08a9b6a1781b39055ecaca1d

### CODE REVIEW

File: conf/layer.conf [hunk 1]
```diff
@@ -10,7 +10,7 @@ BBFILE_PATTERN_qcom := "^${LAYERDIR}/"
 BBFILE_PRIORITY_qcom = "5"
 
 LAYERDEPENDS_qcom = "core"
-LAYERSERIES_COMPAT_qcom = "nanbield"
+LAYERSERIES_COMPAT_qcom = "scarthgap"
```
Comment (lumag) [inline conf/layer.conf:13]: Is it there any reason to drop nanbield?

**Review by lumag — CHANGES_REQUESTED** [review]
- Please provide sensible commit message
- Please drop Change-Id trailer.

**Comment by adhudase** [comment]
> * Please provide sensible commit message
> 
>  * Please drop Change-Id trailer.
Sorry, will update the commit text.

File: conf/layer.conf (hunk 1 above)
Comment (adhudase) [reply conf/layer.conf:13]: No, that was by mistake; will update the commit.

**Pull request merged**